transaction_integration_test.go

```diff
@@ -867,3 +880,56 @@ func TestTransactionCreateWithCustomFields(t *testing.T) {
 		t.Fatalf("Returned custom fields doesn't match input, got %q, want %q", x, customFields)
 	}
 }
+
+func TestTransactionTaxExempt(t *testing.T) {
+	t.Parallel()
+
+	amount := NewDecimal(10000, 2)
+	txn, err := testGateway.Transaction().Create(&TransactionRequest{
+		Type:               "sale",
+		Amount:             amount,
+		TaxExempt:          true,
+		PaymentMethodNonce: FakeNonceTransactable,
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	txn, err = testGateway.Transaction().Find(txn.Id)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	if !txn.TaxExempt {
+		t.Fatalf("Transaction did not return tax exempt")
+	}
+	if txn.TaxAmount.Valid {
```

Comment:
Elsewhere you used `.Valid != true` -- do you want to use `.Valid != false` here?